features/utils/graphql_client.py

```diff
@@ -0,0 +1,36 @@
+from six.moves import urllib
```

Comment:
why use `six.moves` ? Is this ever going to be run under a Python 2 environment ?

Reply:
it's some initial code that I've copied / pasted from some github repo; it will for sure change later on